### ISSUE TITLE
(Bug report) Docker unable to start after updating to 0.61.11

### ISSUE THREAD

**Issue by Prowler1000** [zadam/trilium#4377]
### Trilium Version

0.61.11

### What operating system are you using?

Other (specify below)

### What is your setup?

Local + server sync

### Operating System Version

Docker (Unraid)

### Description

When attempting to start the docker container after updating it to 0.61.11 from 0.60.4, the console is spammed with 

> Saving image 'inline image' as attachment into note 'PfXBE0Jn04bi'

Repeatedly, followed by (the error I pasted into the Error logs section).

The note PfXBE0Jn04bi is one created from an from an imported HTML document from the first day of a lab for a university course I'm taking. I've looked it over and I couldn't find any course specific material but I believe it still falls under copyright so I'll avoid posting it unless it's specifically needed. The document has multiple inline images, and some base64 images.

My Windows 11 clients are still on the previous version as I'd like to avoid losing all my notes if possible, but I _really_ should have been creating backups, I've just been lazy.

If there's any more information needed, please let me know, I'm just trying to throw this together before I head out to class.

### Error logs

> Exception in PromiseRejectCallback:
> /usr/src/app/node_modules/@jimp/core/dist/index.js:734
>   return new Promise((resolve, reject) => {
>-------^
>
> RangeError: Maximum call stack size exceeded
>
> Exception in PromiseRejectCallback:
> /usr/src/app/src/services/image.js:205
>     const image = await jimp.read(buffer);
>-------------------------^
>
> RangeError: Maximum call stack size exceeded
> 
> Saving image 'inline image' as attachment into note 'PfXBE0Jn04bi'
> Transaction rollback dirtied the becca, forcing reload.
> Becca (note cache) load took 3ms
> ERROR: Could not scan for links note 'PfXBE0Jn04bi': Maximum call stack size exceeded RangeError: Maximum call stack size > exceeded
>     at console.value (node:internal/console/constructor:309:13)
>     at console.log (node:internal/console/constructor:380:26)
>     at log (/usr/src/app/src/services/log.js:65:13)
>     at Object.info (/usr/src/app/src/services/log.js:69:5)
>     at Object.saveImageToAttachment (/usr/src/app/src/services/image.js:135:9)
>     at downloadImages (/usr/src/app/src/services/notes.js:531:45)
>     at saveLinks (/usr/src/app/src/services/notes.js:660:19)
>     at Function.<anonymous> (/usr/src/app/src/services/notes.js:875:66)
>     at Function.sqliteTransaction (/usr/src/app/node_modules/better-sqlite3/lib/methods/transaction.js:65:24)
>     at Object.transactional (/usr/src/app/src/services/sql.js:242:52)

![image](https://github.com/zadam/trilium/assets/26493649/fe16c456-0b26-4a67-87fb-86534e45a390)

**Comment by zadam** [comment]
Hi, you should be able to get rid of that by unchecking "Enable image compression":

![image](https://github.com/zadam/trilium/assets/617641/57760b01-1146-4178-b997-d2ffc599fc5a)

**Comment by Prowler1000** [comment]
Is there a way I can edit the config before it starts? I'm unable to access the docker whatsoever (sorry if it's listed in the docs, I didn't see it last time I looked but it's been a minute)

**Comment by zadam** [comment]
I'm currently releasing 0.61.12 which disables the image compression during the migration. Please try that once releases.

**Comment by Prowler1000** [comment]
The issue still seems to persist. I forcefully updated the container, and the repository is set to `zadam/trilium:latest`. Is there any way I can help? Not that I'm confident in how to go about it, but I don't mind packing up all my data and sending it if it would give you an environment to test in. I've been letting the container run for a solid 10 minutes now and it's just the same thing over and over.

If there's nothing I can do to help, is there a way I can delete the problematic note outside of the UI since it's inaccessible?

**Comment by zadam** [comment]
Hello, if you're willing to send me your document.db (to zadam.apps@gmail.com), then this would certainly help. Thanks!

**Comment by Prowler1000** [comment]
Sorry for the late reply, I had a few midterms. In order to sync my files while I prepared, I downgraded the container. 

I went to latest upgrade again one last time to make sure nothing had changed and it seems as though whatever you changed in the 0.61.13 since then has fixed the issue. I was, however, able to cause it to occur again by targeting 0.61.12 and replacing the containers document.db with one from my devices that hasn't been migrated.

Just in case you'd still like to take a look into this, I will send you an email with the un-migrated db file regardless.

Thank you very much for looking into this, I sincerely appreciate the work you do.
